src/scss/components/_badge.scss

```diff
@@ -0,0 +1,4 @@
+.badge {
+  min-width: calc-em(54);
+  line-height: calc-em(22);
```

Comment:
original bootstrap element has line-height too, don't you see a var for that?
or also using its origina padding top/bottom that is 0